tests/testthat/test-score_forecasts.R

```diff
@@ -391,7 +378,56 @@ test_that("abs error is correct, point and median forecasts same", {
 
 })
 
-test_that("wis is correct, median only", {
+test_that("wis is correct, point forecast only, use_median_as_point FALSE",{
```

Comment:
Just for clarity, I suggest calling this test "wis is NaN, point forecast only, use_median_as_point FALSE"